Add Login as Customer inforrmation (#654)
magento/merchdocs: src/customers/account-company-update.md

| Button            | Description |
|-------------------|-------------|
| <span class="btn">Back</span>              | Returns to the Companies page without saving changes. |

> | <span class="btn">Login as Customer</span> | Allows logging in as an admin user to help customers with their orders. |

Reviewer: I think this needs some clarification

```suggestion
| <span class="btn">Login as Customer</span> | Allows an Admin user to log in to the storefront as the customer and help with their orders. |
```
